mypy/main.py

```diff
@@ -940,8 +943,8 @@ def add_invertible_flag(
         description="Adjust the amount of detail shown in error messages.",
     )
     add_invertible_flag(
-        "--show-error-context",
-        default=False,
+        "--hide-error-context",
+        default=True,
```

Comment:
is this changing the default value? these are so hard to read but it looks like its not changing anything?

Reply:
This is correct, it's confusing, but what it means is "the flag is called as '--hide-error-context', it goes to `show_error_context` and the default value to send is `True`. I'm not looking to refactor this stuff when it will conflict heavily with upstream changes.

Thanks for pointing it out though.